src/Instance/MoveHoldingContext/MoveHoldingContext.js

```diff
@@ -215,15 +226,7 @@ const MoveHoldingContext = ({
           id="move-holding-confirmation"
           confirmLabel={intl.formatMessage({ id: 'ui-inventory.moveItems.modal.confirmLabel' })}
           heading={intl.formatMessage({ id: 'ui-inventory.moveItems.modal.title' })}
-          message={
-            intl.formatMessage(
-              { id: 'ui-inventory.moveItems.modal.message' },
-              {
-                count: movingItems.length,
-                targetName: <b>{rightInstance.id === dragToId ? rightInstance.title : leftInstance.title}</b>
-              }
-            )
-          }
+          message={getMovingMessage()}
```

Comment:
this usage semantically works with `useMemo` since you actually don't need a callback